mms/protocol/otf_message_handler.py

```diff
@@ -254,19 +254,19 @@ def _retrieve_input_data(conn):
         return None
 
     model_input = dict()
-    model_input["name"] = _retrieve_buffer(conn, length).decode()
+    model_input["name"] = _retrieve_buffer(conn, length).decode("utf-8")
 
     length = _retrieve_int(conn)
-    content_type = _retrieve_buffer(conn, length).decode()
+    content_type = _retrieve_buffer(conn, length).decode("utf-8")
     model_input["contentType"] = content_type
 
     length = _retrieve_int(conn)
     value = _retrieve_buffer(conn, length)
 
     if content_type == "application/json":
-        model_input["value"] = json.loads(value.decode())
+        model_input["value"] = json.loads(value.decode("utf-8"))
```

Comment:
Should we change the encode() in  "create_load_model_response" to encode("utf-8") as well?